### PR TITLE
GitHub: Update the deploy Action

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

Current delopy pipeline has only deployment pipeline for Preview branch. But I'm gonna add for Production too.

## Task Checklist

- [ ] Added store config/update/submission steps
- [ ] Clean up
- [ ] Apply requested changes

## Known Issues

- Not being able to test on fork branch.

## PR Checklist

- [x] **Closes**: _N/A_
- [x] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Steps To Validate Changes

1. Merge this to the main
2. Run manually
3. Check if succeeded

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -1,10 +1,34 @@
-name: Deploy Preview Pipeline
+# Copyright (c) 2023 Files Community
+# Licensed under the MIT License. See the LICENSE.
+
+# Abstract:
+#  - This CI is executed manually by the maintainer of Files.
+#  - Handles Preview and Store verions to deploy.
+
+name: Files Deploy
```
Comment (yaira2) [inline .github/workflows/deploy.yml:8]:
```suggestion
name: Deploy Files
```

Comment (yaira2) [reply .github/workflows/deploy.yml:8]: Or "Files Deployment"

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+env:
+  WORKING_DIR: ${{ github.workspace }} # Default: 'D:\a\Files\Files'
+  SOLUTION_PATH: '${{ github.workspace }}\Files.sln'
+  PACKAGE_PROJECT_DIR: '${{ github.workspace }}\src\Files.App (Package)'
+  PACKAGE_PROJECT_PATH: '${{ github.workspace }}\src\Files.App (Package)\Files.Package.wapproj'
+  AUTOMATED_TESTS_ARCHITECTURE: 'x64'
+  AUTOMATED_TESTS_CONFIGURATION: 'Release'
+  AUTOMATED_TESTS_PROJECT_DIR: '${{ github.workspace }}\tests\Files.InteractionTests'
+  AUTOMATED_TESTS_PROJECT_PATH: '${{ github.workspace }}\tests\Files.InteractionTests\Files.InteractionTests.csproj'
+  AUTOMATED_TESTS_ASSEMBLY_DIR: '${{ github.workspace }}\artifacts\TestsAssembly'
```
Comment (yaira2) [inline .github/workflows/deploy.yml:22]: Are these needed?

Comment (0x5bfa) [reply .github/workflows/deploy.yml:22]: No, gonna remove

**Pull request closed without merging**